content/en/boilerplates/revision-tags-default-intro.md

```diff
@@ -1,7 +1,11 @@
 ---
 ---
-The revision pointed to by the tag `default` is considered the ***default revision*** and has additional semantic meaning.
+The revision pointed to by the tag `default` is considered the ***default revision*** and has additional semantic meaning. The default
+revision performs the following functions:
 
-The `default` revision will inject sidecars for the `istio-injection=enabled` namespace selector and `sidecar.istio.io/inject=true` object
-selector in addition to the `istio.io/rev=default` selectors. This makes it possible to migrate from using non-revisioned Istio to using
-a revision entirely without relabeling namespaces. To make a revision `1-10-0` the default, run:
+- Injects sidecars with for the `istio-injection=enabled` namespace selector, the `sidecar.istio.io/inject=true` object
+  selector, and the `istio.io/rev=default` selectors
```

Comment:
```suggestion
- Injects sidecars for the `istio-injection=enabled` namespace selector, the `sidecar.istio.io/inject=true` object
  selector, and the `istio.io/rev=default` selectors
```

Does it make sense to say "namespace selector", "object selector" and 'selector" respectively?  Can the latter work on both namespaces and objects?  Are namespaces not considered objects?

Reply:
I kept it namespace selector for "istio-injection" object selector for "sidecar.istio.io/inject" since that's what we support ([here](https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/#controlling-the-injection-policy)), not sure if object selectors _can_ apply to namespaces. `istio.io/rev` is the same for both namespace and object so I just have "selectors" there.